check-unpinned-deps.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs')
+const { join } = require('path')
+
+/**
+ * Scan dependencies in the package.json files in these folders.
+ */
+const folders = ['.', './editor', './utopia-remix', './puppeteer-tests']
+
+const unpinnedDeps = folders.flatMap((projectPath) => {
+  const path = join(projectPath, 'package.json')
+  const packageJson = JSON.parse(fs.readFileSync(path), 'utf8')
+  const allDeps = {
+    ...packageJson.dependencies,
+    // ...packageJson.devDependencies,
+  }
+  return Object.entries(allDeps)
+    .filter(([, version]) => version.match(/^[\^~]/))
```

Comment:
Not sure I understand what the regex is checking, can we not just do `version.startsWith('^')`?

Reply:
yeah it's pointlessly unreadable, updated in https://github.com/concrete-utopia/utopia/pull/4870/commits/190d1ae2c4170ff0fa1b1f24592c782c7af82a82